vault/resource_ldap_secret_backend_library_set_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccLDAPSecretBackendLibrarySet(t *testing.T) {
+	setName := acctest.RandomWithPrefix("tf-test-ldap-library-set")
+	bindDN, bindPass, url := testutil.GetTestLDAPCreds(t)
+	resourceType := "vault_ldap_secret_backend_library_set"
+	resourceName := resourceType + ".set"
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion112)
+		},
+		CheckDestroy: testCheckMountDestroyed(resourceType, consts.MountTypeLDAP, consts.FieldMount),
+		Steps: []resource.TestStep{
+			{
+				Config: testLDAPSecretBackendLibrarySetConfig_defaults(bindDN, bindPass, url, setName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldName, setName),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldServiceAccountNames+".#", "2"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldServiceAccountNames+".0", "bob"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldServiceAccountNames+".1", "alice"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDisableCheckInEnforcement, "false"),
```

Comment:
Should we also check that the defaults from Vault were set for TTL and MaxTTL? The docs say `"24h"` and the TTL was returned as `"86400"` in my local tests